src/main/java/uk/org/tombolo/execution/DataExportEngine.java

```diff
@@ -41,6 +42,13 @@ public void execute(DataExportRecipe dataExportSpec, Writer writer) throws Excep
 	}
 
 	public void execute(DataExportRecipe dataExportSpec, Writer writer, ImporterMatcher forceImports) throws Exception {
+
+		//Check for Valid Providers
+		String vProvider = verifyProvider();
+		if (null != vProvider) {
```

Comment:
typo != -> ==

Reply:
No, this code is actually checking for when provider is not null, if it is null then it mean all providers in recipe are valid and this method is not invoked by test cases, please follow the verifyProvider method again, in case you still find it ambiguous